test-integration/helpers/test-element.coffee

```diff
@@ -0,0 +1,72 @@
+selenium = require 'selenium-webdriver'
```

Comment:
I was wondering if it would make sense to put this in a different directory than the other helpers because they are all expected to be called for doing things on a screen (to avoid confusion)